src/Main/Main.js

```diff
@@ -112,13 +112,13 @@ class Main extends Component {
     vendorCategory: {
       type: 'okapi',
       records: 'categories',
-      path: 'vendor_category'
-    },
-    vendorContactCategory: {
-      type: 'okapi',
-      records: 'categories',
-      path: 'contact_category'
+      path: 'vendor-storage/categories'
     },
+    // vendorContactCategory: {
+    //   type: 'okapi',
+    //   records: 'categories',
+    //   path: 'contact_category'
```

Comment:
I haven't found such API endpoint in https://github.com/folio-org/mod-vendors/pull/43 and I decided to comment it out. The List of vendors and create/edit of specific vendor works well without it, but @bonthebeerman might know better.